src/malcolm/malcolmSocket.test.js

```diff
@@ -12,15 +15,50 @@ describe('malcolm socket handler', () => {
     },
   };
 
-  const socketContainer = new MalcolmSocketContainer(socket);
+  const betterSocket = {
+    onmessage: () => {},
+    onerror: () => {},
+    onopen: () => {},
+    onclose: () => {},
+    send: payload => {
+      messages.push(payload);
+    },
+    isConnected: true,
+  };
 
   beforeEach(() => {
```

Comment:
I'm not sure socketContainer and socketContainerMK2 are the best names here, is there anything more descriptive of why you need 2 different versions?

Reply:
I like this renaming 👍